htmx-spring-boot/src/test/resources/templates/fragments.html

```diff
@@ -2,7 +2,7 @@
 
 <span th:fragment="active-items-count"
       id="active-items-count"
-      hx:swap-oob="${itemCountSwap} ?: ''"
+      th:hx-swap-oob="${itemCountSwap} ?: ''"
```

Comment:
Why did you change this?

Reply:
Since we are not in the thymeleaf module, we cannot use the dialect attributes here.

Reply:
Ah, right.